api/rest.go

```diff
@@ -4190,6 +4320,231 @@ func handleUpdatePublisherI18n(exec boil.Executor, id int64, i18ns []*models.Pub
 	return handleGetPublisher(exec, id)
 }
 
+//Labels
+func handleGetLabelList(exec boil.Executor, r *LabelsRequest) (*LabelsResponse, *HttpError) {
+	mods := make([]qm.QueryMod, 0)
+
+	// filters
```

Comment:
you must filter out label for sensitive / private units if user is not allowed to see such units.
I think inner join with content_units where secure allowed_read(cp)

Reply:
fixed